config/locales/en.yml

```diff
@@ -894,7 +894,7 @@ en:
       teaching_hours_per_week:
         question:
           On average, how many hours per week are you timetabled to teach at %{school_name} during the current term?
-        hint: ‘Teaching’ refers to the direct contact time you spend in lessons with students of all ages.
+        hint: ‘Timetabled teaching hours’ refer to the time you spend teaching lessons to students of all ages.
```

Comment:
think wireframe is wrong and should be `refers` not `refer`. wdyt?

Reply:
yeah, I agree. 
I think the 'refer' is referring to the singular phrase "Timetabled teaching hours", and not the plural hours, so refers is more grammatically correct.